app/src/main/res/anim/fade_out.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The transition looks good, but you could put integer 0 instead of this file (R.anim.fade_out).

Reply:
you can delete this right?